tests/test_connection.py

```diff
@@ -44,3 +43,14 @@ def test_Misc(self):
         result[0].uri['nodelist'].pop()
         self.assertTrue(len(result[0].uri['nodelist']) == 0)
         self.assertEqual("Connection()", repr(self.named_conn))
+
+    @defer.inlineCallbacks
+    def test_uri_input(self):
+        test = txmongo.connection.ConnectionPool()
+        yield test.disconnect()
+        test = txmongo.connection.ConnectionPool("mongodb://127.0.0.1/dbname")
+        yield test.disconnect()
+        test = txmongo.connection.ConnectionPool(u"mongodb://127.0.0.1/dbname")
```

Comment:
Under Python 3 these two checks will be the same. May be use `b"..."` for the first one to test byte-string too? Or we are not going to support bytes in Py3?